### DISCUSSION TITLE
form component for realtime preview?

### DISCUSSION THREAD

**Discussion by Saifallak** [pboivin/filament-peek#26] [Ideas]
form component for realtime preview?

**Comment by pboivin** [comment]
Hi @Saifallak. This is not quite the realtime preview, but I'm currently working on something similar for version 1.0 of the plugin. It introduces a new mode for the preview modal designed for the Builder field (can also work with any other fields). You can get your form and the preview side-by-side in the modal, with some support for automatically refreshing the preview when the form is updated. I wrote a bit of documentation on it, more will come in the next week or so :

https://github.com/pboivin/filament-peek/blob/1.x/docs/builder-field-previews.md
